Source: git://git.debian.org/git/pkg-xorg/lib/mesa
commit 012d959a18d229601de040d8d557572330809eb9
Author: Julien Cristau <jcristau@debian.org>
Date:   Tue Jun 25 21:33:41 2013 +0200

    Make sure libgbm doesn't have undefined symbols.

diff --git a/src/gbm/Makefile.am b/src/gbm/Makefile.am
index e22c55c..556eb03 100644
--- a/src/gbm/Makefile.am
+++ b/src/gbm/Makefile.am
@@ -18,7 +18,7 @@ libgbm_la_SOURCES = \
 	main/gbm.c \
 	main/backend.c \
 	main/common.c
-libgbm_la_LDFLAGS = -version-info 1:0
+libgbm_la_LDFLAGS = -version-info 1:0 -Wl,-z,defs
 libgbm_la_LIBADD = $(LIBUDEV_LIBS) $(LIBKMS_LIBS) $(DLOPEN_LIBS)
 
 if HAVE_EGL_PLATFORM_WAYLAND
@@ -39,6 +39,8 @@ libgbm_dri_la_CFLAGS = \
 
 libgbm_la_LIBADD += \
 	libgbm_dri.la $(top_builddir)/src/mapi/shared-glapi/libglapi.la
+
+libgbm_la_LDFLAGS += $(LIBDRM_LIBS)
 endif
 
 all-local: libgbm.la
